### PR TITLE
added autoversioning and corrected publish task

### PR DESCRIPTION
**Pull request by vkkhare**

### CODE REVIEW

**Comment by vkkhare** [comment]
Closes #99

**Review by karlhigley — APPROVED** [review]

File: jvm/publish.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.jfrog.bintray'
 apply plugin: 'com.github.dcendents.android-maven'
 
-version = '0.0.8'
+version = System.getenv("GITHUB_REF")
```
Comment (gmuraru) [inline jvm/publish.gradle:4]: Q: Should it be better to have another env variable "GITHUB_REF" and get the version from that one?
Also, is ```GITHUB_REF``` the release version (in this context)?

Comment (vkkhare) [reply jvm/publish.gradle:4]: yeah GITHUB_REF is the release version as specified by @vvmnnnkv

**Comment by vkkhare** [comment]
Also, I would like to wait before merging this PR. I also need to add compile options, I realised just now.

**Comment by karlhigley** [comment]
@vkkhare Is this ready to merge? (I think it might be, but not sure if you've had a chance to address the compile options mentioned above.)

**Comment by vkkhare** [comment]
I have added the change @karlhighley You can merge it

**Pull request merged**